src/sass/_permission_group_selection.scss

```diff
@@ -0,0 +1,51 @@
+.modified-actions {
+  align-items: center;
+  display: flex;
+  gap: $sph--large;
+  justify-content: flex-end;
+  margin-bottom: calc(0.75 * $spv--large);
+
+  .modified-status {
+    align-items: center;
+    display: flex;
+    gap: $sp-unit;
+
+    span {
+      color: $color-mid-dark;
+    }
+  }
+}
+
+#group-selection-table {
+  .selected-row {
+    background-color: white;
+  }
```

Comment:
this was done intentionally after discussion with @piperdeck . Since the highlighting would be in effect as long as the box is checked, this may interfere with the visual representation of the modification status.